Source Code/Ticket Management System/supportgiver_menu.c

```diff
@@ -77,143 +77,157 @@ bool searchByTags(void* ticket, void* tag)
 	return found_tag;
 }
 
+void filterByStatusAction() 
+{
+	struct Vector* tickets = ticketContainer_getAll();
+	if (tickets == NULL) {
+		wprintf(L"No tickets in the system.\n");
```

Comment:
Even though there are no tickets, it will continue executing this function & ask for status.